charts/matrix-stack/configs/matrix-rtc/sfu/config-underrides.yaml.tpl

```diff
@@ -0,0 +1,30 @@
+{{- /*
+Copyright 2024-2025 New Vector Ltd
+
+SPDX-License-Identifier: AGPL-3.0-only
+*/ -}}
+
+{{- $root := .root -}}
+{{- with required "matrix-rtc/sfu/config.yaml.tpl missing context" .context -}}
+
+rtc:
+  use_external_ip: true
+
+prometheus_port: 6789
+
+# Logging config
+logging:
+  # log level, valid values: debug, info, warn, error
+  level: {{ .logging.level }}
+  # log level for pion, default error
+  pion_level: {{ .logging.pionLevel }}
+  # when set to true, emit json fields
+  json: {{ .logging.json }}
```

Comment:
This should all be in overrides. Any additional logging settings will still get merged in